spec/controllers/summary_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+describe SummaryController do
+  describe "#index" do
```

Comment:
should we test the case where someone visits `index/:last_year` since that is an option?